src/EA.Weee.RequestHandlers/Admin/GetActiveComplianceYears/GetAatfReturnsActiveComplianceYearsDataAccess.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace EA.Weee.RequestHandlers.Admin.GetActiveComplianceYears
+{
+    using System.Collections.Generic;
+    using System.Data.Entity;
+    using System.Linq;
+    using System.Threading.Tasks;
+    using DataAccess;
+
+    public class GetAatfReturnsActiveComplianceYearsDataAccess : IGetAatfReturnsActiveComplianceYearsDataAccess
+    {
+        private readonly WeeeContext context;
+
+        public GetAatfReturnsActiveComplianceYearsDataAccess(WeeeContext context)
+        {
+            this.context = context;
+        }
+
+        public async Task<List<int>> Get()
+        {
+            return await context.Returns
```

Comment:
Add data access tests and tests to ensure the return data is ordered correctly

Reply:
Stephen has the tests for this code on his branch.